Ontologies.Mappings/test/Ontologies.Mappings.Test.csproj

```diff
@@ -28,8 +28,7 @@
 		<PackageReference Include="Moq" Version="4.18.2" />
 		<PackageReference Include="MSTest.TestAdapter" Version="2.2.10" />
 		<PackageReference Include="MSTest.TestFramework" Version="2.2.10" />
-		<PackageReference Include="WillowInc.Ontology.Airport.DTDLv3" Version="1.0.0.5" />
-		<PackageReference Include="WillowInc.Ontology.DTDLv3" Version="1.0.1.33" />
```

Comment:
@joebeernink With `WillowInc.Ontology.DTDLv3` removed, how do we manage the version of the Willow ontology in our tests?